src/fidesops/migrations/versions/55d61eb8ed12_add_default_policies.py

```diff
@@ -0,0 +1,213 @@
+    ).first()
+
+    if not client:
+        logger.info(f"No autogenerated client: '{FIDESOPS_AUTOGENERATED_CLIENT_KEY}'")
+        return
+
+    access_policy = Policy.filter(
+        db=db,
+        conditions=(
+            Policy.key == AUTOGENERATED_ACCCES_KEY and Policy.client_id == client.id
+        ),
+    ).first()
+    if access_policy:
+        logger.info(
+            f"Deleting autogenerated '{AUTOGENERATED_ACCCES_KEY}' access policy"
+        )
+        # Only delete "download" policy if it was created by the autogenerated client
+        access_policy.delete(db)
+
+    erasure_policy = Policy.filter(
+        db=db,
+        conditions=(
+            Policy.key == AUTOGENERATED_ERASURE_KEY and Policy.client_id == client.id
+        ),
+    ).first()
+
+    if erasure_policy:
+        # Only delete "delete" policy if it was created by the autogenerated client
+        logger.info(
+            f"Deleting autogenerated '{AUTOGENERATED_ERASURE_KEY}' erasure policy"
+        )
+        erasure_policy.delete(db)
+
+    try:
+        logger.info(
+            f"Deleting autogenerated client: '{FIDESOPS_AUTOGENERATED_CLIENT_KEY}'"
+        )
+        client.delete(db)
+        storage = StorageConfig.filter(
+            db=db, conditions=(StorageConfig.key == FIDESOPS_AUTOGENERATED_STORAGE_KEY)
+        ).first()
+        if storage:
+            logger.info(
+                f"Deleting autogenerated local storage: '{FIDESOPS_AUTOGENERATED_STORAGE_KEY}'"
+            )
+            storage.delete(db)
+    except Exception:
+        # It's possible the client or storage config have been attached to other things
+        pass
+
+    db.close()
+
+
+def autogenerate_access_policy(
+    db: Session, client: ClientDetail, storage: StorageConfig
+) -> None:
+    """Create an autogenerated 'download' access policy, with an access rule attached,
+    targeting user.provided.identifiable data"""
+    logger.info(f"Creating autogenerated '{AUTOGENERATED_ACCCES_KEY}' policy")
+    access_policy = Policy.create_or_update(
+        db,
+        data={
+            "name": "Fidesops Autogenerated Access Policy",
+            "key": AUTOGENERATED_ACCCES_KEY,
+            "drp_action": DrpAction.access,
+            "client_id": client.id,
+        },
+    )
+
+    access_rule = Rule.create(
+        db,
+        data={
+            "name": "Fidesops Autogenerated Access Rule",
+            "action_type": ActionType.access.value,
+            "policy_id": access_policy.id,
+            "client_id": client.id,
+            "storage_destination_id": storage.id,
+        },
+    )
+
+    RuleTarget.create(
+        db=db,
+        data={
+            "name": "Fidesops Autogenerated Access Target",
+            "data_category": DataCategory("user.provided.identifiable").value,
+            "rule_id": access_rule.id,
+            "client_id": client.id,
+        },
+    )
```

Comment:
I tried to use sane defaults for these, local storage which we already discussed, and targeting identifiable data, happy to adjust.

Reply:
This is great, thanks @pattisdr